src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/CSharpExpressionCompilerTest.csproj

```diff
@@ -112,10 +112,13 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="PerfTests\DebuggerSteppingTestcase.xml" />
```

Comment:
We probably need a "CopyToOutput" property on these (you can click on the file in VS and set this property to "Copy if newer" or whatever the other PerfTests have...